docs/config/retriever_config.md

```diff
@@ -97,3 +97,48 @@ def convert(
         xr.DataArray: The converted DataArray for the specified variable.
     """
 ```
+
+## Coords
+
+The coords section in the retriever.yaml file specifies the coordinate variables that are used to read and interpret raw input data. This section defines the dimensions and attributes of the coordinates, such as time or spatial coordinates, which are essential for accurately mapping the data within the dataset. The configuration ensures that the data is properly aligned and can be processed effectively by the tsdat framework
```

Comment:
"Coords" should be "Coordinates" for readability

"retriever.yaml" should be \`retriever.yaml\` so it shows up nicely.